twilio/http/http_client.py

```diff
@@ -94,10 +88,30 @@ def request(self, method, url, params=None, data=None, headers=None, auth=None,
             timeout=timeout if timeout is not None else self.timeout,
         )
 
-        self.logger.info('{method} Response: {status} {text}'.format(
-            method=method, status=response.status_code, text=response.text)
-        )
+        self._log_response(response)
 
         self.last_response = Response(int(response.status_code), response.text, response.headers)
 
         return self.last_response
+
+    def _log_request(self, method, params, kwargs, url, headers):
```

Comment:
`method`, `params`, `url`, and `headers` are included in `kwargs` so they can be removed from method signature.

Reply:
Ahh yes, fixed.